.drone.yml

```diff
@@ -6106,12 +6892,39 @@ steps:
     "$ARTIFACT_PATH"
   environment:
     ARTIFACT_PATH: /go/artifacts
-    AWS_ACCESS_KEY_ID:
-      from_secret: AWS_ACCESS_KEY_ID
     AWS_S3_BUCKET:
       from_secret: AWS_S3_BUCKET
```

Comment:
Correct me if I'm wrong here but doesn't this step (along with the "upload" step further down) need to include the `awsconfig` volume? Otherwise how does this step get the AWS credentials from "Assume Download AWS Role"?

Reply:
I believe both have the `awsconfig` volume.  See line 6897-6899 for download and 6962-6963 for upload.  It i easier to see outside of side-by-side mode, where the diff algo gives poor presentation.

Reply:
100% right! I just can't read today.